delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/resolvers/ResolverState.scala

```diff
@@ -0,0 +1,179 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model.resolvers
+
+import java.time.Instant
+
+import cats.data.NonEmptyList
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.{nxv, schemas}
+import ch.epfl.bluebrain.nexus.delta.sdk.ResolverResource
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ResourceRef.Latest
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.resolvers.Resolver.{CrossProjectResolver, InProjectResolver}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{AccessUrl, ResourceF, ResourceRef}
+
+/**
+  * Enumeration of Resolver state types
+  */
+sealed trait ResolverState extends Product with Serializable {
+
+  /**
+    * @return the schema reference that resolvers conforms to
+    */
+  final def schema: ResourceRef = Latest(schemas.resolvers)
+
+  /**
+    * @return the collection of known types of resolvers resources
+    */
+  final def types: Set[Iri] = Set(nxv.Schema)
```

Comment:
why there is a `nxv.Schema` here?

The types should be: nxv.Resolver plus `nxv.CrossProject` or `nxv.InProject`